src/ws/lobby.js

```diff
@@ -11,66 +11,74 @@ function find(connections, sessions, ws, participantId, sessionId) {
         }
     }
 
-    if (participantId && sessionId) {
-        const session = sessions.get(sessionId);
-
-        if (session) {
-            const connection = session.get(participantId);
-
-            if (connection) {
-                return [connection, participantId, sessionId];
+    if (participantId && sessionAlias && game) {
+        const gameSession = gameSessions.get(game);
+        if (gameSession) {
+            const session = gameSession.get(sessionAlias);
+            if (session) {
+                const connection = session.get(participantId);
+                if (connection) {
+                    return [connection, participantId, sessionAlias, game];
```

Comment:
Why is this an array and not an object?
`{ connection, participant, sessionAlias, game }`

Reply:
Well, because the whole FE service works with such arrays. There's no background for it, "jist because" :)

Reply:
It looks like C 🐱 
Not good, not bad, just looks like C to me

Reply:
Can we make constants for the indices then?

Reply:
What names?

Reply:
`connection`, `participantId`, ...

Reply:
But how are numbers better than names?

Reply:
`generic`ticity

Reply:
Okay, but the info data elements are still distinguished by their roles if not by content, right?
Like 0 is apparently a connection key, 1 is session key and so on
Can't we find better, more readable names than number for them?

Reply:
Or the roles can be different too?